src/screens/CreatePost/index.js

```diff
@@ -561,27 +560,6 @@ const CreatePost = () => {
     );
   };
 
-  const handleTextMessage = () => {
-    if (!typeUser) {
-      return `There are new posts from ${user.myProfile.username} & others`;
-    }
-    return 'There are new posts from Anonymous & others';
-  };
-
-  const handleTopicChat = async () => {
-    const allTopics = listTopic;
-
-    allTopics.forEach(async (topic) => {
-      const channel = await client.client.channel('topics', `topic_${topic}`, {
-        name: `#${topic}`,
-        members: [user.myProfile.user_id],
-        channel_type: 3
-      });
-      await channel.create();
-      await channel.addMembers([user.myProfile.user_id]);
-      await channel.sendMessage({text: handleTextMessage()}, {skip_push: true});
-    });
-  };
   const createPoll = () => {
     setIsPollShown(true);
     sheetMediaRef.current.close();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the `handleTopicChat` function and related logic seems to be a significant change. This function was responsible for creating channels and sending messages based on topics. Now, it's removed without any replacement. If this functionality is no longer needed, then the changes are fine. However, if similar functionality is expected elsewhere, ensure that it's implemented properly.


<!-- This is an auto-generated comment by OSS CodeRabbit -->